pm64/src/bgm/midi.rs

```diff
@@ -407,6 +422,21 @@ fn midi_track_to_bgm_track(
                                             },
                                         );
                                     }
+                                    100 => {
+                                        if pitch_range_cmd_state == 1 {
```

Comment:
If you do this
```suggestion
                                    100 if pitch_range_cmd_state == 1 => {
```
You can do
`_ => pitch_range_cmd_state = 0`